src/utils/transittool.rsc

```diff
@@ -86,117 +87,88 @@ Macro "TransitScenarioComparison" (Args, S2_Dir, TOD)
     for dir in scen_Dirs do
         i = i + 1
         skim_dir_transit = dir + "\\output\\skims\\transit"
-        //skim_dir_walk = dir + "\\output\\skims\\nonmotorized"
         parts = ParseString(dir, "\\")
         scen_name = parts[parts.length]
-        //trn_skim_files = RunMacro("Catalog Files", {dir: skim_dir_transit, ext: "mtx"})
         
+        // Create a matrix to store results (only do this once)
         trn_skim_file = skim_dir_transit + "/skim_" + TOD + "_w_all.mtx"
-        // Create a starting transit matrix to store results (only do this once)
         if i = 1 then do
-            out_file = tod_dir + "/TransitShortestPath_" + scen_name + "_" + TOD + ".mtx"
+            out_file = tod_dir + "/Comparisons_" + "_" + TOD + ".mtx"
             CopyFile(trn_skim_file, out_file)
             mtx = CreateObject("Matrix", out_file)
             core_names = mtx.GetCoreNames()
-            mtx.AddCores({"Temp"})
+            mtx.AddCores({"temp"})
             mtx.DropCores(core_names)
         end
 
-        // Create cores for each scenario to store SP and total trips
-        mtx.AddCores({scen_name + "_w_skim"})
+        //loop through all transit modes under walk access
+        out_trip_core = null
+        out_skim_core = null
+        out_wttime_core = null
         mtx.AddCores({scen_name + "_trips"})
-        out_skim_core = mtx.GetCore(scen_name + "_w_skim")
         out_trip_core = mtx.GetCore(scen_name + "_trips")
-        out_skim_core := 0
-        out_trip_core := 0
-        
-        skim_mtx = CreateObject("Matrix", trn_skim_file)
-        skim_core = skim_mtx.GetCore("Total Time")
-        //out_skim_core := if nz(skim_core)<=60 then nz(skim_core) else 0
-        out_skim_core := nz(skim_core)
-
-        //Open transit trip mtx and sum up cores to SP mtx to get total trips OD
-        trip_Dir = dir + "\\output\\\assignment\\transit"
-        trip_file = trip_Dir + "/transit_" + TOD + ".mtx"
-        trip_mtx = CreateObject("Matrix", trip_file)
-        trip_core_names = trip_mtx.GetCoreNames()
-        for trip_core_name in trip_core_names do
-            trip_core = trip_mtx.GetCore(trip_core_name)
-            out_trip_core := out_trip_core + nz(trip_core)
+        mtx.AddCores({scen_name + "_skims"})        
+        out_skim_core = mtx.GetCore(scen_name + "_skims")
```

Comment:
Replace all calls to GetCore() with the shorthand/preferred approach. See the following example:

   folder = RunMacro("G30 Tutorial Folder")
   mobj1 = CreateObject("Matrix", folder + "Amarillo PA.mtx")
   mobj2 = CreateObject("Matrix", folder + "Amarillo AutoCost.mtx")
   mobj1.AddCores({"VHT"})
   mobj1.VHT := mobj1.Total * mobj2.Time / 60 // matrix operation using core names to calculate vht

On this line of your code, for example, you can just do: 

out_skim_core = mtx.(scen_name + "_skims")

Or you could get rid of the out_skim_core extra variable completely. Up to you.

You don't need to replace any old cold (Stuff I wrote before I built this new approach), but any new code going forward (including this PR) should adopt the canonical (and easier!) way of getting cores.

Reply:
Thanks for the guidance! That saves so many lines.